BCP195bis/draft-ietf-uta-rfc7525bis.md

```diff
@@ -341,6 +341,9 @@ These are minimum recommendations for the use of TLS in the vast majority of imp
 
 Community knowledge about the strength of various algorithms and feasible attacks can change quickly, and experience shows that a Best Current Practice (BCP) document about security is a point-in-time statement.  Readers are advised to seek out any errata or updates that apply to this document.
     
+This document updates {{RFC5288}} and {{RFC6066}} in view of the recent {{Boeck2016}} and {{ALPACA}} attacks.
+See {{nonce-reuse}} and {{sni}} for the details.
+
```

Comment:
I think this is *too* concise. These are two separate updates, each worthy of a separate sentence.

Reply:
Done in [fc5b1f5](https://github.com/yaronf/I-D/pull/432/commits/fc5b1f5e1c498292deff69c57f4293a3c4385c84)